src/com/company/Problem_statement_1.java

```diff
@@ -0,0 +1,49 @@
+package com.company;
```

Comment:
Sorting related handling for words is not done in the code. 
There can be multiple URLs in the url.txt file. 
Please read the problem statement again and update the code.

Reply:
Handled the sorting part.
It can work for multiple urls.